### ISSUE TITLE
fix(engines): syncpack 12.x supports node >=18 not >=16

### ISSUE THREAD

**Issue by 350str** [JamieMason/syncpack#215]
## Description

Hello. I found a problem, syncpack v12 doesn't work with node v16, because packages globby v14+ and ora v8+ require node v18+

## Suggested Solution

update engines field in package.json for correct node version (>18) or downgrade major version of this packages in deps

**Comment by JamieMason** [comment]
Thanks a lot @350str, I'll bump `engines` in the next release. By the way, if you are on Node 16 there are older versions of syncpack which work with that.

**Comment by 350str** [comment]
yea, v11 works great, thanks!

**Comment by lkraav** [comment]
Yikes, and here I am on Node 16, needing `--specs` released in 12.2.0 :crying_cat_face: :

**Comment by JamieMason** [comment]
https://github.com/JamieMason/syncpack/releases/tag/14.0.0-alpha.1 might work if you want to try it @lkraav

**Comment by JamieMason** [comment]
Can you summarise what you were wanting to do please @lkraav? To find and fix something specific by the sound of it?

**Comment by lkraav** [comment]
> Can you summarise what you were wanting to do please @lkraav? To find and fix something specific by the sound of it?

Certainly. I was looking to take control of very limited amount of package globs at a time.

It's unclear from alpha release notes if syncpack-14 even supports `.syncpackrc` anymore?

Beyond that, I would've wanted to run `syncpack fix --filter "koa"` like the good ol days, because I can't blow up the repo by upgrading 100 packages at once.

**Comment by JamieMason** [comment]
Thanks this is helpful.

> It's unclear from alpha release notes if syncpack-14 even supports `.syncpackrc` anymore?

Config file loading and filenames are the same as before, some config values have changed a little though - I need to write a lot of docs very soon.

--filter is replaced by --dependencies, you can see _some_ examples here 
https://x.com/fold_left/status/1852349762596417967?t=HvGGFZdeHphFvcbpldXjtg&s=19 and also in the --help output

> Beyond that, I would've wanted to run `syncpack fix --filter "koa"` like the good ol days, because I can't blow up the repo by upgrading all 100 packages at once.

I changed filtering to only apply to linting in 14-alpha but this has changed my mind, I'll put that back in the fix command so you can do this

**Comment by JamieMason** [comment]
Closing as complete and have created #263 for the filtering we discussed at the end.
